cli/crates/cli/tests/init_ts.rs

```diff
@@ -0,0 +1,45 @@
+mod utils;
+
+use backend::project::ConfigType;
+use serde_json::json;
+use utils::environment::Environment;
+
+#[test]
+fn init_ts() {
+    let env = Environment::init();
+
+    env.write_json_file_to_project(
+        "package.json",
+        &json!({
```

Comment:
We could maybe make this a file, not critical

Reply:
It's nice that you see the before and after in the same test.

Reply:
Ok